app/Http/Controllers/PostLikeController.php

```diff
@@ -8,39 +8,21 @@
 
 class PostLikeController extends Controller
 {
-
```

Comment:
Ovo sam ti prepisao, ako je neko.  lajkovao onda na isti klik odlajkuje i sklonio sam onaj response 409, to niko nikad nije koristio :)